frontend/src/__tests__/cypress/cypress/pages/projects.ts

```diff
@@ -95,6 +95,26 @@ class ProjectListPage {
   findCreateWorkbenchButton() {
     return cy.findByRole('button', { name: 'Create a workbench' });
   }
+
+  findProjectsTypeDropdown() {
+    return cy.findByLabelText('Options menu');
+  }
+
+  findProjectsTypeDropdownByText(projectType: string) {
+    return cy.contains('button', projectType);
+  }
+
+  findProjectsDropdownFilter(){
+    return cy.findByTestId('filter-dropdown-select');
```

Comment:
I believe you can use `.findFilterMenuOption('filter-dropdown-select', 'Name')` as we have custom commands to work with dropdowns.

Reply:
It worked [fd7be8c](https://github.com/opendatahub-io/odh-dashboard/pull/3028/commits/fd7be8c15bf8ddea1712bc40d7b9caba6cd2f53e) 
Thanks!